source/amazon/services/supported-services/ecr-image-scanning.rst

```diff
@@ -0,0 +1,114 @@
+.. Copyright (C) 2021 Wazuh, Inc.
+
+.. _amazon_image_scanning:
+
+Amazon ECR Image scanning
+=========================
+
+.. versionadded:: 4.3.0
+
+`Amazon ECR image scanning <https://docs.aws.amazon.com/AmazonECR/latest/userguide/image-scanning.html>`_ uses the Common Vulnerabilities and Exposures (CVEs) database from the open-source `Clair project <https://github.com/quay/clair>`_ to detect software vulnerabilities in container images and provide a list of scan findings, which can be easily integrated into Wazuh thanks to the :ref:`AWS CloudWatch Logs integration <aws_cloudwatchlogs>`.
+
+Amazon ECR sends an event to Amazon EventBridge when an image scan is completed. The event itself is only a summary and does not contain the details of the scan findings. However, it is possible to configure a Lambda function to request the scan findings details and store them in CloudWatch Logs. Here is a quick resume on how the workflow looks like:
```

Comment:
I think that you misused 'resume' which is a false friend. You could use either 'résumé' or 'summary'.
```suggestion
Amazon ECR sends an event to Amazon EventBridge when an image scan is completed. The event itself is only a summary and does not contain the details of the scan findings. However, it is possible to configure a Lambda function to request the scan findings details and store them in CloudWatch Logs. Here is a quick summary of how the workflow looks like:
```

Reply:
Done